site/book-theme/header.hbs

```diff
@@ -110,6 +110,108 @@
   padding: 0.25em 0;
   white-space: nowrap;
 }
+
+.main-nav__item:focus {
+  outline: 1px dashed #fff;
+  outline-offset: 0.3em;
+}
+
+.main-nav__button-container {
+  display: none;
+  height: 71px;
+  width: 71px;
+  cursor: pointer;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#menu-toggle {
+  display: none;
+}
+
+.main-nav__button,
+.main-nav__button::before,
+.main-nav__button::after {
+  display: block;
+  background-color: #fff;
+  position: absolute;
+  height: 4px;
+  width: 30px;
+  transition: transform 400ms cubic-bezier(0.23, 1, 0.32, 1);
+  border-radius: 2px;
+}
+
+.main-nav__button::before {
+  content: '';
+  margin-top: -8px;
+}
+
+.main-nav__button::after {
+  content: '';
+  margin-top: 8px;
+}
+
+#menu-toggle:checked + .main-nav__button-container .main-nav__button::before {
+  margin-top: 0px;
+  transform: rotate(405deg);
```

Comment:
This animation is cool but a bit excessive.

Reply:
It's not uncommon for hamburger menus to have some animation. Would you prefer to remove it and just change from the hamburger to the X?